template.tpl

```diff
@@ -169,6 +221,7 @@ const Math = require('Math');
  */
 const dlp = createQueue('dataLayer');
 function dataLayerPush(payload) {
+  dlp({ ecommerce: null });
```

Comment:
Clearing the datalayer before each ecommerce call is apparently a GTM best practice.